webapp/sources/rudder/rudder-web/src/main/scala/com/normation/rudder/web/components/RuleGrid.scala

```diff
@@ -233,7 +219,7 @@ class RuleGrid(
               ruleCompliances = {};
               recentChanges = {};
               recentGraphs = {};
-              refreshTable("${htmlId_rulesGridId}", ${newData.json.toJsCmd});
+              refreshTable("${htmlId_rulesGridId}", ${newData});
```

Comment:
for now, `newData` returns a string but we will rewrite it with `toJson` from zio

Reply:
as in a preview comment, `${newData}` will use the default `toString` on a case class, `JsTableLine` here, so it's not a valid json, we should still keep the `newData.json.toJsCmd`

Reply:
ah yes, I changed return to `JsTableLine` back since comment, and forgot to correct all places